### PR TITLE
Django Cheat Sheet: Various improvements

### PR DESCRIPTION
**Pull request by 122131**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.
- [ ] New Instant Answer
  - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
  - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
  - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
  - [x] Enhancement: **`{Django Cheat Sheet}: {Various updates}`**
- [ ] Non–Instant Answer
  - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**
###### Description of changes
- Removed tabbing
- Updated partially according to #2779 
###### People to notify (@mention interested parties)

@pjhampton @GuiltyDolphin @moollaza 

---

Instant Answer Page: https://duck.co/ia/view/django_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @rsunder10

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by 122131** [comment]
I have never used Django, so I don't know if this is correct or if this cheat sheet could be made in a different way...
If you have any suggestion feel free to ask! 😄

**Comment by pjhampton** [comment]
Hey @122131

First of all, thanks so much for contributing! 😄  I really appreciate it. It's great to have so many Python enthusiasts contributing to DuckDuckHack.

I have had a look over the PR. While I certainly agree that improvements could be made both in terms of content and aesthetics to this cheat sheet, I'm not sure if you have addressed these. For example, you have removed the field lookups which is actually really helpful for a django developer because the ORM is quite big and complicated. Also in #3366 3b4bc76df0f5878c264eaf521de0a24da2eec7fe you change the aliases to a more verbose alternative which, I believe would lead to less relevant triggering.

I think you should roll back some of your changes and resubmit with just the description. I believe this is much better than the original and then we can regroup to discuss the other improvements you can make to this cheat sheet. Are you on the slack channel? My moniker is @pjhampton as usual.

For future reference, I always think you can add the most value to DuckDuckHack cheat sheets if you are familiar with the technology or actively learning it. If you're not too sure, please head on over to https://forum.duckduckhack.com/c/programming/python and discuss with us so we can help you make the maximum impact.

Thanks again 💛  💜  ❤️

**Comment by 122131** [comment]
@pjhampton No problem! 👍 
I'll revert the changes except the description and (eventually) the source URL (if it is ok).
Thanks to you for the suggestions! 😃

File: share/goodie/cheat_sheets/json/django.json [hunk 1]
```diff
@@ -1,96 +1,139 @@
...
+    "metadata": {
+        "sourceName": "Django Project",
+        "sourceUrl": "https://docs.djangoproject.com/"
+    },
 
-	"aliases": [
-		"pydjango", "python django","django lookups","django admin commands","django queryset commands","django query"
-	],
+    "aliases": [
+        "pydjango", "python django","django admin commands","django manage.py commands","django-admin"
```
Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/django.json]: @122131 I will suggest keeping the `"django lookups"` and `"django query"` aliases as well.

**Comment by GuiltyDolphin** [comment]
@rsunder10 Have you had a chance to look over the new changes?

**Comment by pjhampton** [comment]
Closing as this PR looks to have gone stale.

If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Pull request closed without merging**